src/main/java/com/codeborne/selenide/SelenideElement.java

```diff
@@ -923,26 +995,27 @@
    * Check if image is properly loaded.
    *
    * @throws IllegalArgumentException if argument is not an "img" element
-   * @since 2.13
-   *
    * @see com.codeborne.selenide.commands.IsImage
+   * @since 2.13
    */
   boolean isImage();
 
   /**
    * Take screenshot of this element
-   * @return file with screenshot (*.png)
    *
+   * @return file with screenshot (*.png)
    * @see com.codeborne.selenide.commands.TakeScreenshot
    */
+  @Nonnull
```

Comment:
AFAIK it is Nullable 
see https://github.com/selenide/selenide/blob/master/src/main/java/com/codeborne/selenide/impl/ScreenShotLaboratory.java#L110